sopel/config/types.py

```diff
@@ -236,17 +266,22 @@ def configure(self, prompt, default, parent, section_name):
             prompt = prompt[0]
 
         if default is not NO_DEFAULT:
-            default = ','.join(default)
-            prompt = '{} [{}]'.format(prompt, default)
+            default_prompt = ','.join(map(lambda item: '"{}"'.format(item), default))
```

Comment:
I would rather see a list-comprehension than a `map` + `lambda`:

    ','.join(['"{}"'.format(item) for item in default])

Reply:
:+1: